jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+pipeline {
+    agent { node { label 'jenkins-workers-slow' } }
+
```

Comment:
can you add discard and scheduler in your jenkinsfile?

https://github.com/Azure/sonic-build-tools/blob/master/jenkins/mellanox/buildimage-mlnx-201904/Jenkinsfile

Reply:
```
    options {

        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '30'))



    }



    triggers {

        pollSCM('@midnight')

    }
```

Reply:
added